packages/contracts-core/contracts/test/Home.t.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.6.11;
+
+import {Home} from "../Home.sol";
+import {NomadTestWithUpdaterManager} from "./utils/NomadTest.sol";
+import {IUpdaterManager} from "../interfaces/IUpdaterManager.sol";
+import {Message} from "../libs/Message.sol";
+
+
+contract HomeTest is NomadTestWithUpdaterManager {
+    Home home;
+
+    function setUp() public override {
+        super.setUp();
+        home = new Home(homeDomain);
+        home.initialize(IUpdaterManager(address(updaterManager)));
+        updaterManager.setHome(address(home));
+    }
+
+    function test_homeDomain() public {
+        assertEq(
+            keccak256(abi.encodePacked(homeDomain, "NOMAD")),
+            home.homeDomainHash()
+        );
+    }
+
+    function test_onlyUpdaterManagerSetUpdater() public {
+        vm.prank(address(updaterManager));
+        home.setUpdater(vm.addr(420));
```

Comment:
future: add some expected reverts to these permissions tests? 

i.e. test both that the manager CAN and that other addresses CANNOT